### ISSUE TITLE
Feature request: RelaxedOneHotCategorical distribution

### ISSUE THREAD

**Issue by lkhphuc** [deepmind/distrax#122]
TFD: https://www.tensorflow.org/probability/oryx/api_docs/python/oryx/distributions/RelaxedOneHotCategorical

**Comment by wbrenton** [comment]
Is there still interest in a gumbel softmax?
